parbase/FairContFact.h

```diff
@@ -56,6 +56,7 @@ class FairContFact : public TNamed
     FairParSet* getContainer(const char*);
     virtual FairParSet* createContainer(FairContainer*) {return 0;}
     virtual void activateParIo(FairParIo*) {}
+    Bool_t AddContainer(FairContainer*);
```

Comment:
Please add a doxygen comment to clarify ownership semantics of the raw pointer, e.g.:

```suggestion
    /// @param[in] container Transfers ownership if return value is true
    Bool_t AddContainer(FairContainer* container);
```